### DISCUSSION TITLE
¿ a donde subimos lo de la pasteleri?

### DISCUSSION THREAD

**Discussion by RupertoTacuche** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#245] [Q&A] [answered]
Hola amigos. ¿ a donde subimos lo de las pasteleria?. Un cordial saludo.

**Comment by AngelCruzO** [comment]
Aun no se publica el forms, tranquilo y no desesperes

**Comment by JosePabloGrr** [comment, marked as answer]
Teams > Semana 2 - Primer Vuelo > Link de entrega Semana 2.
![image](https://user-images.githubusercontent.com/16726985/156844897-1d98640b-949d-40c9-9ec0-2b57017cc3da.png)

**Reply by doguedogue** [reply]
https://forms.office.com/r/NUEk6AKqvD

**Reply by JuanSalvador73** [reply]
Gracias por la info :D

**Comment by RupertoTacuche** [comment]
GRacias
